backupccl: skip flaky backup partitioned test

pbardea opened the pull request:
This test is flaking due to inconsistency in placing data on all the
nodes. Skipping until resolved to stabilize master.

Informs https://github.com/cockroachdb/cockroach/issues/64974.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/65055)
<!-- Reviewable:end -->

adityamaru commented:
I _think_ we lint usages of `t.Skip` and recommend using skip.WithIssue(...). You might already know this but we have `bin/skip-test` that does this automatically for us.

pbardea committed:
backupccl: skip flaky backup partitioned test

This test is flaking due to inconsistency in placing data on all the
nodes. Skipping until resolved to stabilize master.

Release note: None

pbardea commented:
Updated. I just wanted to skip the subtest and from what I saw `bin/skip-test` doesn't support grepping for those.

pbardea commented:
bors r=adityamaru
